src/assets/styles/theme.scss

```diff
@@ -67,3 +67,11 @@ $app-blue: #1669bb;
 $app-dk-blue: #38598a; // same as the Vuetify theme appDkBlue
 $app-red: #d3272c;
 $app-green: #1a9031;
+
+// Sizes for Fonts
```

Comment:
i like it. give us a base to work with.  and some sanity too.

Reply:
Thanks!

And visibility, too. And it's much easier to search/reference variables than values :)